ide-common/src/main/kotlin/org/digma/intellij/plugin/editor/PaginationUtil.kt

```diff
@@ -0,0 +1,36 @@
+package org.digma.intellij.plugin.editor
+
+import java.util.concurrent.ConcurrentHashMap
+
+private val insightsPaginationInfo: ConcurrentHashMap<String, ConcurrentHashMap<String, Int>> = ConcurrentHashMap()
+
+fun addInsightPaginationInfo(focusedDocumentName: String, uniqueInsightId: String, activePage: Int) {
+    if (activePage > 1) {
+        val insightsPaginationInfosMapForActualFile = insightsPaginationInfo.get(focusedDocumentName)
+        if (insightsPaginationInfosMapForActualFile != null) {
+            insightsPaginationInfosMapForActualFile[uniqueInsightId] = activePage
+            insightsPaginationInfo[focusedDocumentName] = insightsPaginationInfosMapForActualFile
```

Comment:
Is this line really needed?
`insightsPaginationInfo[focusedDocumentName] = insightsPaginationInfosMapForActualFile`

Reply:
it's not needed as well :)